components/runners/ambient-runner/ambient_runner/middleware/grpc_push.py

```
@@ -0,0 +1,127 @@
+            yield event
+        return
+
+    try:
+        async for event in event_stream:
+            yield event
+            _push_event(grpc_client, sid, event)
+    finally:
+        try:
+            grpc_client.close()
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)